spec/commands/show_spec.rb

```diff
@@ -30,7 +30,7 @@
       expect(out).to eq(default_bundle_path("gems", "rails-2.3.2").to_s)
     end
 
-    it "prints deprecation", :bundler => "2" do
+    it "prints deprecation", :bundler => "<= 2" do
```

Comment:
Since we have added a `< 3` filter to the outermost block, I think this is redundant and the `:bundler` filter can be removed.

Reply:
Same with the rest of `:bundler` filters in this file.